src/network/utilization.rs

```diff
@@ -3,14 +3,15 @@ use crate::network::{Connection, Direction, Segment};
 use ::std::collections::HashMap;
 
 #[derive(Clone)]
-pub struct TotalBandwidth {
+pub struct ConnectionInfo {
+    pub interface: String,
```

Comment:
Can we use the `NetworkInterface` type here? Not a biggy, but will help my obsessive heart. :)

Reply:
Sure, but don't you think it would be a bit expensive to copy the `NetworkInterface` for every message when only the name is needed? Or maybe you were thinking about wrapping it in an `Arc<Mutex<>>` ?

Reply:
Hmm... tbh I'm a little iffy about the way this is stringified now. But it indeed doesn't make a lot of sense, you're right. Let's leave it like this (or change the name if you feel it would be better).

Reply:
So I've changed the name to `interface_name` to be more explicit.

To try to make my point a bit more clear, I think it's fair here to take one property of the `NetworkInterface` (its name) and pass it to the UI to display it. In this case we are not interested in e.g. the `flags` field (Operating system specific flags for the interface).

Reply:
I understand the point perfectly, no worries. I was mostly thinking that I don't like the way the connection string is displayed in the UI at the moment (I suggested it, so totally on me :) ) - and might want to change it to something else later, or just sort it by interface and not display, or... I dunno. But if we do that we can change this, so all's good. :)

Reply:
I see now I completely missed your point :) I understand now